addons/website_sale/models/product_product.py

```diff
@@ -160,6 +159,20 @@ def _to_markup_data(self, website):
         """
         self.ensure_one()
 
+        product_price = request.pricelist._get_product_price(
+            self, quantity=1, target_currency=website.currency_id,
+        )
+        product_taxes = self.sudo().taxes_id._filter_taxes_by_company(self.env.company)
+        taxes = request.fiscal_position._value.map_tax(product_taxes)
+        price = self.product_tmpl_id._apply_taxes_to_price(
+            price=product_price,
+            currency=website.currency_id,
+            product_taxes=product_taxes,
+            taxes=taxes,
+            product_or_template=self,
+            website=website,
```

Comment:
Since it args, not kwargs
```suggestion
            product_price, website.currency_id, product_taxes, taxes, self, website=website,
```